src/views/v2/emails/template-parts/body/order/order-total.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Event Tickets Emails: Order Total
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/emails/template-parts/body/order/order-total.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var Tribe_Template  $this  Current template object.
+ * @var Module           $provider              [Global] The tickets provider instance.
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var \WP_Post         $order                 [Global] The order object.
+ * @var int              $order_id              [Global] The order ID.
+ * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
+ */
+
+ // @todo @codingmusician @juanfra Replace hardcoded data with dynamic data.
+
+?>
+<tr>
+	<td style="padding-top:20px;text-align:right" align="right">
+		<table style="display:inline-block;width:auto">
+			<tr>
+				<td style="font-size:14px;font-weight:400;line-height:24px;padding-right:10px">Order Total</td>
```

Comment:
We should move as much of these inline styles to the `/v2/emails/template-parts/header/head/styles.php` template as CSS classes, so that's easier for people to customize these, instead of going through a bunch of templates. 

that way presentation is separate from content, and they can adjust whatever padding or something they like from the styles, and if they want to alter the content they go to the template itsel.

Reply:
The reason I chose to do inline styling is that is what my research suggested when I was looing into email design. I'd much rather use stylesheets. I'll go ahead and do it for now, but we need to thoroughly test different email clients to ensure everything looks okay.

Reply:
You can always check what's supported [here](https://www.caniemail.com/)